ArduPlane/quadplane.cpp

```diff
@@ -2050,11 +2050,21 @@ bool QuadPlane::verify_vtol_takeoff(const AP_Mission::Mission_Command &cmd)
  */
 void QuadPlane::check_land_complete(void)
 {
+    uint32_t now = AP_HAL::millis();
+
+    if (poscontrol.state == QPOS_LAND_COMPLETE) {
+        if (plane.landing.get_disarm_delay() <= 0 ||
+            (now - landing_detect.lower_limit_start_ms >= plane.landing.get_disarm_delay()*1000UL)) {
+            // disarm immediately or after user-defined delay
+            plane.disarm_motors();
```

Comment:
i think we need to keep the load() of airspeed_cruise_cm, otherwise landing missions with airspeed changes won't reset correct airspeed